straxen/plugins/online_monitor.py

```diff
@@ -178,6 +179,82 @@ def area_width_hist(self, data):
         return hist.T
 
 
+@export
+class OnlineSEMonitor(strax.Plugin):
+    """
+    Plugin to write data needed for the online SE monitor to the 
+    online-monitor collection in the runs-database. Data that is written by
+    this plugin should be small such as to not overload the runs-
+    database. If the peaks are large, random
+    max_bytes of data are selected from the peaks.
+
+    This plugin takes 'peak_basics' and 'peak_positions_mlp'. Although 
+    they are not strictly related, they are aggregated into a single data_type
+    in order to minimize the number of documents in the online monitor.
+
+    Produces 'online_se_monitor' with info on the peaks and their
+    positions.
+    """
+
+    max_bytes = straxen.URLConfig(
+        default=10e6,
+        track=True,
+        help='Maximum amount of bytes of data for MongoDB document'
+    )
+
+    depends_on = ('peak_basics', 'peak_positions_mlp')
+    provides = 'online_se_monitor'
+    data_kind = 'online_se_monitor'
```

Comment:
I think you can call it something more general than SE monitor, the plugin does monitor much more than just SE, what do you think